### PR TITLE
revert packr files

### PR DESCRIPTION
**Pull request by tchughesiv**
@ruromero @bmozaffa @swati-kale are we sure these packr files are necessary? if so, we need add them back.

Signed-off-by: tchughesiv <tchughesiv@gmail.com>

### CODE REVIEW

**Comment by ruromero** [comment]
I understood they are needed for including the static resources into the generated binary.

**Comment by ruromero** [comment]
I have just tested it and it is included without the need of using packr. So I understand we can even remove the packr dependency at all. Can't we?

**Comment by bmozaffa** [comment]
Basically the entirety of this project is consumed through packr. That's the case when running this project standalone with Go, but not when running it in dev mode with node.

From the ba-operator, packr files would be used to read all html/css/js. At least as of a few months ago, our packr version did not have the ability to generate packr files inside its vendor directory. Furthermore, the convention with other library projects is also to commit the packr files.

In short, yes, we should keep the packr files here.

**Comment by ruromero** [comment]
I ran the console binary built from the operator using the version that doesn’t have the packr go files and didn’t miss any static resource.
How can this be?

**Comment by bmozaffa** [comment]
Were the source files in the same directory? That works explain it.

Relatedly, it doesn't work in the operator deployment either: https://console-cr-form-babak-operator-ui.apps.mw-ocp4.cloud.lab.eng.bos.redhat.com/

**Comment by tchughesiv** [comment]
@bmozaffa @ruromero so we should merge this?

**Comment by bmozaffa** [comment]
Yes!
